fe/fe-core/src/main/java/org/apache/doris/nereids/cost/CostModelV1.java

```diff
@@ -271,14 +271,14 @@ public Cost visitPhysicalDistribute(
         Statistics childStatistics = context.getChildStatistics(0);
         double intputRowCount = childStatistics.getRowCount();
         DistributionSpec spec = distribute.getDistributionSpec();
-
+        int beNumForDist = Math.max(3, beNumber);
```

Comment:
let this local var be a CostModelV1's member var?

Reply:
this is used for distribute and join. Other SQL operator still use beNumber. I think it is better to make it function scope